apple_passkit/lib/src/apple_pk_pass.dart

```diff
@@ -61,13 +61,42 @@ class ApplePkPass {
 }
 
 /// Status after trying to add multiple passes.
-enum PkPassType {
+enum PKPassType {
   /// A nonspecific pass type.
-  any,
+  any('18446744073709551615'),
 
   /// A pass that represents a barcode.
-  barcode,
+  barcode('0'),
 
   /// A pass that represents a credential that the device stores in the Secure Element.
-  secureElement
-}
+  secureElement('1'),
+
+  /// Deprecated, but maps to secureElement value
+  payment('1');
+
+  /// Constructor to initialize each enum case with a string representation of the integer value
+  const PKPassType(this.value);
+
+  /// A field to store the string value associated with each enum case (representing integers as strings)
+  final String value;
+
+  // Factory method to create an enum from a JSON value (string)
+  static PKPassType fromJson(String jsonValue) {
+    for (PKPassType type in PKPassType.values) {
+      if (type.value == jsonValue) {
+        return type;
+      }
+    }
+    throw ArgumentError('Invalid PKPassType value: $jsonValue');
```

Comment:
What do you think of falling back to `any` instead of throwing?

Reply:
I think it is a good solution, it removes the necessity for exception handling.